napari/components/dims.py

```diff
@@ -40,22 +40,23 @@ class Dims(EventedModel):
     last_used : int
         Dimension which was last used.
     range : tuple of 3-tuple of float
-        List of tuples (min, max, step), one for each dimension. In a world
-        coordinates space. As with Python's `range` and `slice`, max is not
-        included.
-    current_step : tuple of int
-        Tuple the slider position for each dims slider, in slider coordinates.
+        List of tuples (min, max, step), one for each dimension in world
+        coordinates space.
+    span : tuple of 3-tuple of float
+        Tuple of (low, high) bounds of the currently selected slice in world space.
     order : tuple of int
         Tuple of ordering the dimensions, where the last dimensions are rendered.
     axis_labels : tuple of str
         Tuple of labels for each dimension.
+    current_step : tuple of int
+        Tuple the slider position for each dims slider, in world coordinates.
```

Comment:
```suggestion
        The slider positions for each dims slider, in world coordinates.
```

Alternatively, `Tuple of the slider ...`. But I'm getting sick of reading `tuple` in these docstrings 😛